tests/modules/enroll_activity/app/test_enroll_activity_presenter.py

```diff
@@ -0,0 +1,404 @@
+                     "time": "12/Mar/2020:19:03:58 +0000",
+                     "timeEpoch": 1583348638390
+              },
+              "body": '{"user_id": "db43", "code": "ECM2345"}',
+              "pathParameters": None,
+              "isBase64Encoded": None,
+              "stageVariables": None
+              }
+
+              response = lambda_handler(event, None)
+              assert response["statusCode"] == 403
+              assert json.loads(response["body"]) == 'That action is forbidden for this Enrollment'
+
+       def test_enroll_activity_presenter_404_no_items_found_activity(self):
+              event = {
+              "version": "2.0",
+              "routeKey": "$default",
+              "rawPath": "/my/path",
+              "rawQueryString": "parameter1=value1&parameter1=value2&parameter2=value",
+              "cookies": [
+                     "cookie1",
+                     "cookie2"
+              ],
+              "headers": {
+                     "header1": "value1",
+                     "header2": "value1,value2"
+              },
+              "queryStringParameters": {
+                     'query_params': "value1"
+              },
+              "requestContext": {
+                     "accountId": "123456789012",
+                     "apiId": "<urlid>",
+                     "authentication": None,
+                     "authorizer": {
+                     "iam": {
+                            "accessKey": "AKIA...",
+                            "accountId": "111122223333",
+                            "callerId": "AIDA...",
+                            "cognitoIdentity": None,
+                            "principalOrgId": None,
+                            "userArn": "arn:aws:iam::111122223333:user/example-user",
+                            "userId": "AIDA..."
+                     }
+                     },
+                     "domainName": "<url-id>.lambda-url.us-west-2.on.aws",
+                     "domainPrefix": "<url-id>",
+                     "external_interfaces": {
+                     "method": "POST",
+                     "path": "/my/path",
+                     "protocol": "HTTP/1.1",
+                     "sourceIp": "123.123.123.123",
+                     "userAgent": "agent"
+                     },
+                     "requestId": "id",
+                     "routeKey": "$default",
+                     "stage": "$default",
+                     "time": "12/Mar/2020:19:03:58 +0000",
+                     "timeEpoch": 1583348638390
+              },
+              "body": '{"user_id": "80fb", "code": "CODIGO_INEXISTENTE"}',
+              "pathParameters": None,
+              "isBase64Encoded": None,
+              "stageVariables": None
+              }
+
+              response = lambda_handler(event, None)
+              assert response["statusCode"] == 404
+              assert json.loads(response["body"]) == 'No items found for Activity'
```

Comment:
Quando for fazer o presenter, podem somente colocar **_test_<nome da rota>_presenter_<nome do status code http>_**

ex: test_enroll_activity_presenter_bad_request

Provavelmente vcs copiaram esse teste do drop_activity e adaptaram pra vcs. O Soller é psicopata, n precisa criar tantos testes pra presenter não kkkkkk O mínimo q eu gostaria seria um teste pra cada tipo (possível de ser retornado) de status code http. Nesse caso, 2 para Created (201) (já que foram criados enrollments no banco de dados, sendo que IN_QUEUE tem uma json.loads(response["body"])['message'] diferente de ENROLLED), uma pra BadRequest (400), uma pra Forbidden(403) e uma pra NotFound(404)